pkg/apis/crd/v1alpha1/types.go

```diff
@@ -350,14 +350,27 @@ type Rule struct {
 	// +optional
 	Ports []NetworkPolicyPort `json:"ports,omitempty"`
 	// Rule is matched if traffic originates from workloads selected by
-	// this field. If this field is empty, this rule matches all sources.
+	// this field. If this field is empty and this is an ingress rule,
+	// this rule matches all sources.
 	// +optional
 	From []NetworkPolicyPeer `json:"from"`
 	// Rule is matched if traffic is intended for workloads selected by
-	// this field. If this field is empty or missing, this rule matches all
-	// destinations.
+	// this field. This field can't be used with ToService. If this field
+	// and ToService are both empty or missing and this is an egress rule,
+	// this rule matches all destinations.
 	// +optional
 	To []NetworkPolicyPeer `json:"to"`
+	// Rule is matched if traffic is intended for workloads selected by
+	// the Service listed on Service ports. Currently only ClusterIP and
+	// NodePort types Services are supported in this field.
```

Comment:
you need to update the comment given that we dropped NodePort from this PR

Reply:
Done.